bugbug/repository.py

```diff
@@ -288,23 +288,29 @@ def hg_modified_files(hg, commit):
     )
 
 
-def get_touched_functions(path, deleted_lines, added_lines, content):
-    if content is None:
-        return set()
+def get_functions_from_metrics(metrics_space):
+    functions = []
+
+    if metrics_space["kind"] == "function" and metrics_space["name"] != "<anonymous>":
```

Comment:
Previously, we were skipping entries with `error` == `True`, are there such entries in the metrics results too?

Reply:
In case of error the space name is empty:
https://github.com/mozilla/rust-code-analysis/blob/master/src/metrics.rs#L105

Reply:
OK, so I'll skip spaces with no name

Reply:
Done with d2e4bae.